terraform/application/storage_account.tf

```diff
@@ -0,0 +1,37 @@
+resource "azurerm_storage_account" "general_storage_account" {
+  count                            = var.create_storage_account ? 1 : 0
+  name                             = var.storage_account_name
+  resource_group_name              = local.app_resource_group_name
+  location                         = "UK South"
+  account_tier                     = "Standard"
+  account_replication_type         = var.account_replication_type
+  allow_nested_items_to_be_public  = false
+  cross_tenant_replication_enabled = false
+
+  dynamic "blob_properties" {
+    for_each = var.environment == "production" ? [1] : []
+
+    content {
+      delete_retention_policy {
+        days = 7
+      }
+
+      container_delete_retention_policy {
+        days = 7
+      }
+    }
+  }
+
+  tags = {
+    environment = var.environment
+  }
```

Comment:
Shouldn't need this tag as it should be added automatically from the resource group

Reply:
I used https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/terraform/aks/storage_account.tf as my reference. I don't actually know what this does 😅 

Can I drop lines 25:27?

Reply:
yes you can drop 25:27